Source/JavaScriptCore/wasm/WasmBBQPlan.cpp

```diff
@@ -178,7 +178,6 @@ void BBQPlan::work(CompilationEffort effort)
 
     computePCToCodeOriginMap(context, linkBuffer);
 
-    dataLogLnIf(shouldDumpDisassemblyFor(CompilationMode::BBQMode), "Generated BBQ code for WebAssembly BBQ function[", m_functionIndex, "] ", signature.toString().ascii().data(), " name ", makeString(IndexOrName(functionIndexSpace, m_moduleInformation->nameSection->get(functionIndexSpace))).ascii().data());
     dumpDisassembly(context, linkBuffer);
```

Comment:
Isn't the above needed for this `BBQPlan::dumpDisassembly` call?  According to `BBQPlan::dumpDisassembly`, it only dumps if `!Options::useSinglePassBBQJIT()`.  Hence, it only looks redundant because we're running with the `Options::useSinglePassBBQJIT()`.  Instead, you should move the `dataLogLnIf` above into `BBQPlan::dumpDisassembly`.

Reply:
No. LinkBuffer's dump generates the same string.